src/devices/acurite.c

```diff
@@ -468,17 +470,23 @@ static int acurite_txr_callback(bitbuffer_t *bitbuf) {
 	    sensor_status = bb[2]; // @todo, uses parity? & 0x07f
 	    humidity = acurite_getHumidity(bb[3]);
 	    tempc = acurite_txr_getTemp(bb[4], bb[5]);
-	    tempf = celsius2fahrenheit(tempc);
-
-	    printf("%s Acurite tower sensor 0x%04X Ch %c: %3.1F C %3.1F F %d %% RH\n",
-		   time_str, sensor_id, channel, tempc, tempf, humidity);
-
-	    // currently 0x44 seens to be a normal status and/or type
-	    // for tower sensors.  Battery OK/Normal == 0x40
-	    if (sensor_status != 0x44)
-		printf("%s Acurite tower sensor 0x%04X Ch %c, Status %02X\n",
-		       time_str, sensor_id, channel, sensor_status);
+            sprintf(channel_str, "%c", channel);
+            battery_low = sensor_status >>7;
+		
+        data = data_make(
+                "time",			"",		DATA_STRING,	time_str,
+                "model",		"",		DATA_STRING,	"Acurite tower sensor",
+                "id",			"",		DATA_INT,	sensor_id,
+                "channel",  		"",     	DATA_STRING, 	&channel_str,
+                "temperature_C", 	"Temperature",	DATA_FORMAT,	"%.1f C", DATA_DOUBLE, tempc,
+                "humidity",		"Humidity",	DATA_INT,	humidity,
+                "battery",        	"Battery",    	DATA_STRING, 	battery_low ? "LOW" : "OK",
```

Comment:
Also, per some of the discussion we had, I'd would output the raw status byte (in addition to battery) in case there are other status bits/message type indications that aren't being parsed yet.   This allows a consumer with more acurite specific implementation to do something with it.